src/wrappers/ThermalProbe_TC.cpp

```diff
@@ -1,10 +1,22 @@
 #include "wrappers/ThermalProbe_TC.h"
 
+#if defined(ARDUINO_CI_COMPILATION_MOCKS)
+#define _GLIBCXX_TYPE_TRAITS 1
+#define _GLIBCXX_CMATH 1
+#if __APPLE__
+#define _GLIBCXX_NUMERIC_LIMITS 1
+#endif
+#endif
+
+#include "model/Statistic.h"
 #include "model/TC_util.h"
 #include "wrappers/DateTime_TC.h"
 #include "wrappers/EEPROM_TC.h"
 #include "wrappers/Serial_TC.h"
 
+//  local statistics
+Statistic myStats;
```

Comment:
Could we name this something like "thermalSample" or "uncorrectedThermalSample"? And perhaps put it in the header file?

Reply:
Renamed and moved to header.